packages/github/data_stream/issues/fields/fields.yml

```
@@ -6,20 +6,16 @@
       fields:
         - name: id
           type: integer
-          description: >-
-            The id of github issue
+          description: The id of github issue.
```

```suggestion
          description: The id of GitHub issue.
```
(similar throughout for human-directed text)

---

Updated in [54a4fed](https://github.com/elastic/integrations/pull/11518/commits/54a4fedaad0639396ab329a6c6d965ee1d580738).